### PR TITLE
[zero] add more models and optimize computing model size

### PR DESCRIPTION
**Pull request by ver217**

### CODE REVIEW

File: zero/colossalai_utils/gpt2_config.json [hunk 1]
```diff
@@ -4,8 +4,8 @@
         "type": "gpt2_10b"
     },
     "hyperparameter": {
-        "batch_size": 20,
-        "steps_per_epoch": 10
+        "batch_size": 1,
```
Comment (feifeibear) [inline zero/colossalai_utils/gpt2_config.json:7]: why 1?

Comment (ver217) [reply zero/colossalai_utils/gpt2_config.json:7]: To test max model size.

**Comment by kurisusnowdeng** [comment]
The following old files are not useful any more. Could you help delete them?

- [colossalai_utils/zero2_config.json](https://github.com/hpcaitech/ColossalAI-Benchmark/blob/77b1370b0e0c1ec748bd3e03f167e0c9ab0efea8/zero/colossalai_utils/zero2_config.json)
- [colossalai_utils/zero3_config.json](https://github.com/hpcaitech/ColossalAI-Benchmark/blob/77b1370b0e0c1ec748bd3e03f167e0c9ab0efea8/zero/colossalai_utils/zero3_config.json)
- [common/gpt.py](https://github.com/hpcaitech/ColossalAI-Benchmark/blob/77b1370b0e0c1ec748bd3e03f167e0c9ab0efea8/zero/common/gpt.py)
- [common/model.py](https://github.com/hpcaitech/ColossalAI-Benchmark/blob/77b1370b0e0c1ec748bd3e03f167e0c9ab0efea8/zero/common/model.py)
- [deepspeed_utils/config.json](https://github.com/hpcaitech/ColossalAI-Benchmark/blob/77b1370b0e0c1ec748bd3e03f167e0c9ab0efea8/zero/deepspeed_utils/config.json)
- [fairscale_utils/config.json](https://github.com/hpcaitech/ColossalAI-Benchmark/blob/77b1370b0e0c1ec748bd3e03f167e0c9ab0efea8/zero/fairscale_utils/config.json)
- [patrickstar_utils/config.json](https://github.com/hpcaitech/ColossalAI-Benchmark/blob/77b1370b0e0c1ec748bd3e03f167e0c9ab0efea8/zero/patrickstar_utils/config.json)
- [torch_utils/config.json](https://github.com/hpcaitech/ColossalAI-Benchmark/blob/77b1370b0e0c1ec748bd3e03f167e0c9ab0efea8/zero/torch_utils/config.json)

**Comment by ver217** [comment]
> The following old files are not useful any more. Could you help delete them?
> 
> * [colossalai_utils/zero2_config.json](https://github.com/hpcaitech/ColossalAI-Benchmark/blob/77b1370b0e0c1ec748bd3e03f167e0c9ab0efea8/zero/colossalai_utils/zero2_config.json)
> * [colossalai_utils/zero3_config.json](https://github.com/hpcaitech/ColossalAI-Benchmark/blob/77b1370b0e0c1ec748bd3e03f167e0c9ab0efea8/zero/colossalai_utils/zero3_config.json)
> * [common/gpt.py](https://github.com/hpcaitech/ColossalAI-Benchmark/blob/77b1370b0e0c1ec748bd3e03f167e0c9ab0efea8/zero/common/gpt.py)
> * [common/model.py](https://github.com/hpcaitech/ColossalAI-Benchmark/blob/77b1370b0e0c1ec748bd3e03f167e0c9ab0efea8/zero/common/model.py)
> * [deepspeed_utils/config.json](https://github.com/hpcaitech/ColossalAI-Benchmark/blob/77b1370b0e0c1ec748bd3e03f167e0c9ab0efea8/zero/deepspeed_utils/config.json)
> * [fairscale_utils/config.json](https://github.com/hpcaitech/ColossalAI-Benchmark/blob/77b1370b0e0c1ec748bd3e03f167e0c9ab0efea8/zero/fairscale_utils/config.json)
> * [patrickstar_utils/config.json](https://github.com/hpcaitech/ColossalAI-Benchmark/blob/77b1370b0e0c1ec748bd3e03f167e0c9ab0efea8/zero/patrickstar_utils/config.json)
> * [torch_utils/config.json](https://github.com/hpcaitech/ColossalAI-Benchmark/blob/77b1370b0e0c1ec748bd3e03f167e0c9ab0efea8/zero/torch_utils/config.json)

DONE

**Review by kurisusnowdeng — APPROVED** [review]

**Pull request merged**